src/icons/logo-header.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="33" height="32" viewBox="0 0 33 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
After some testing I think adding new logo file makes new sensy as the old one was OK. The size difference could be fixed by changing CSS.